readthedocs/api/v3/urls.py

```diff
@@ -77,6 +78,17 @@
     ],
 )
 
+# allows /api/v3/projects/pip/versions/v3.6.2/filetreediff/
+versions.register(
```

Comment:
This API should be under /projects/{project}/filetreediff, as both versions should be passed as parameters.